pcicrawler/cli.py

```diff
@@ -316,6 +610,10 @@ def main(
         no_scripting()
         # When asked to print a tree, include filtered devices parents
         print_tree(sorted(devs, key=lambda d: d.device_name))
+    elif ocp:
+        run = tv.TestRun(name="pcicrawler", version="")
```

Comment:
version could be the pcicrawler module version here

Reply:
yeah bug, need to fix :)